apps/settings/locales/settings.en-US.properties

```diff
@@ -325,11 +325,11 @@ cellularAndData=Cellular & Data
 # If you need to adapt your localization to the reduced space available
 # in headers, you can use a shorter string here.
 cellularAndData-header={{cellularAndData}}
-dataNetwork=Carrier
+dataOperator=Operator
 dataConnectivity=Data
 dataConnection=Data Connection
 dataRoaming=Data Roaming
-networkOperator=Network Operator
+netOperator=Operator
```

Comment:
there's a string that use networkOperator
`networkOperator-header={{networkOperator}}`